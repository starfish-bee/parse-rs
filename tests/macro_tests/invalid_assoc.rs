use parser::*;

#[derive(Operator, Debug, Clone, Copy)]
enum Op {
    #[assoc("up")]
    #[ident("#")]
    Hash,
    #[assoc("right")]
    #[ident("^")]
    Caret,
    #[ident("?")]
    QMark,
}

fn main() {}
