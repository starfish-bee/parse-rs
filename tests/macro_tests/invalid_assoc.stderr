error: expected 'left', 'right', 'prefix' or 'postfix'
 --> $DIR/invalid_assoc.rs:5:13
  |
5 |     #[assoc("up")]
  |             ^^^^
